sources/editor/Stride.Assets.Presentation/Templates/ModelAssetTemplateWindow.xaml

```diff
@@ -19,14 +19,15 @@
   </Window.Resources>
   <StackPanel>
     <Border Background="{StaticResource EmphasisColorBrush}" Margin="20,20,20,0">
-      <TextBlock Text="{sd:Localize Materials and textures}" Margin="20,10" FontSize="16"/>
+      <TextBlock Text="{sd:Localize Materials textures and animations}" Margin="20,10" FontSize="16"/>
```

Comment:
Careful when charging text that is localized. The localization framework is based on the exact string (it serves as a key), so any previous translations will stop working.

Alternatively, an English translation file can be created (if not already the case) and update the text there, while keeping the original key.

On there other hand, if that particular text wasn't translated yet, then it's fine to change it.